docs/development.md

```diff
@@ -177,6 +177,7 @@ func (c *Client) GetImageStreams(namespace string) ([]imagev1.ImageStream, error
 
   You can see the entire test function `TestGetImageStream` in [pkg/occlient/occlient_test.go](https://github.com/redhat-developer/odo/blob/master/pkg/occlient/occlient_test.go)
 
+**NOTE**: You can use environment variable `CUSTOM_HOMEDIR` to specify a custom home directory to use. It is mainly meant to be used in environments where a user and his home directory is unresolveable. Set the variable, in such cases and then run the tests.
```

Comment:
You can use environment variable `CUSTOM_HOMEDIR` to specify a custom home directory. It can be used in environments where a user and home directory are not resolveable.

Reply:
Going for a more concise and gender neutral message ;-)

Reply:
Makes sense, updating

Reply:
@girishramnani updated